### ISSUE TITLE
initialRegion not working on Android

### ISSUE THREAD

**Issue by grigored** [airbnb/react-native-maps#1559]
Tried rendering the example for initialRegion. Works fine on ios, but on android the map is zoomed out and centered on (0, 0) 
```
<MapView
    initialRegion={{
      latitude: 37.78825,
      longitude: -122.4324,
      latitudeDelta: 0.0922,
      longitudeDelta: 0.0421,
    }}
  />
```

**Comment by christopherdro** [comment]
@grigored What version are you on?

**Comment by christopherdro** [comment]
https://github.com/airbnb/react-native-maps/commit/fd8dc08ffaa0fadbaa73d9e29033a16f808e318f Could be the cause of the issue.
If you're on 0.16.1 can you try 0.16.0 and see if it works?

**Comment by grigored** [comment]
I had a look at that pr before posting this issue, but i didn't think it can be the cause. Just tested with 0.16.0, and it's the same
<img width="755" alt="screen shot 2017-08-16 at 8 19 31 pm" src="https://user-images.githubusercontent.com/12827406/29376187-6c5821e8-82c0-11e7-8bde-cf59165d1582.png">

**Comment by grigored** [comment]
I originally had 0.16.1

**Comment by grigored** [comment]
I'm sorry, I used "^0.16.0" and that actually installed 0.16.1. I tested again and I can confirm on 0.16.0 android is working fine, so https://github.com/airbnb/react-native-maps/pull/1546 broke it.

**Comment by christopherdro** [comment]
@pellepersson Was this the breaking change you were referring too?

**Comment by foyarash** [comment]
I've got this problem too, I may have an idea on how to repair my PR but I can only work on it by tomorrow.

**Comment by christopherdro** [comment]
Ok keep me posted tomorrow. If not I'll revert and do  a new release until we can figure it out.

**Comment by foyarash** [comment]
@christopherdro should be fixed now in https://github.com/airbnb/react-native-maps/pull/1563 , works for me now

**Comment by grigored** [comment]
I just tested both Android and iOS and it works fine! Thanks for the quick fix @foyarash

**Comment by foyarash** [comment]
@grigored no problem, that was my mistake so I had to fix it as fast as possible :)

**Comment by grigored** [comment]
@foyarash , i just noticed there's another bug, but this one happens on 0.16.0 as well: on android, when using initialregion, the map is first centered on 0,0 and zoomed out, as if initialregion was not defined, and then after 2 seconds it centers on the defined initialregion.

My code is as in the example:
```
<MapView
    initialRegion={{
      latitude: 37.78825,
      longitude: -122.4324,
      latitudeDelta: 0.0922,
      longitudeDelta: 0.0421,
    }}
  />
```

**Comment by aleyakovle** [comment]
@grigored I think InitialRegion sets to true coordinates after second render. Becouse If I change Mob-X state, map reloads good, with true coordinates.

**Comment by kubido** [comment]
same with me.. I replace `initialRegion` with `Region`

**Comment by christopherdro** [comment]
Should be fixed in https://github.com/airbnb/react-native-maps/releases/tag/v0.16.2

**Comment by grigored** [comment]
Yes it is fixed in 0.16.2. I ended up making my component controlled, and using `region` (not initialRegion), and `onRegionChangeComplete={region => this.setState({region})}`.
